packages/bolt-integration/src/bolt-button-strategy.ts

```diff
@@ -100,20 +116,67 @@ export default class BoltButtonStrategy implements CheckoutButtonStrategy {
         boltButtonContainer.setAttribute('style', 'display:none');
         boltButtonContainer.setAttribute('data-tid', 'product-page-checkout-wrapper');
 
-        boltButtonObject.setAttribute('data', this.getBoltObjectData(paymentMethod));
+        boltButtonObject.setAttribute('data', this.getBoltObjectData(paymentMethod, style));
         boltButtonObject.setAttribute('class', 'bolt-product-checkout-button');
 
         boltButtonContainer.append(boltButtonObject);
         container.innerHTML = '';
         container.append(boltButtonContainer);
     }
 
-    private getBoltObjectData(paymentMethod: BoltPaymentMethod): string {
+    private getBoltObjectData(
+        paymentMethod: BoltPaymentMethod,
+        style?: BoltButtonStyleOptions,
+    ): string {
         const { initializationData, config } = paymentMethod;
         const { publishableKey, developerConfig } = initializationData || {};
 
         const domainUrl = this.boltScriptLoader.getDomainURL(!!config.testMode, developerConfig);
+        const buttonHeight = this.getButtonHeight(style?.size);
+        const buttonBorderRadius = this.getButtonBorderRadius(style?.shape, buttonHeight);
+        const urlParameters = stringify({
+            publishable_key: publishableKey,
+            variant: 'ppc',
+            ...(buttonHeight && { height: buttonHeight }),
+            ...(buttonBorderRadius && { border_radius: buttonBorderRadius }),
+        });
+
+        return `https://${domainUrl}/v1/checkout_button?${urlParameters}`;
+    }
+
+    private getButtonHeight(buttonSize?: StyleButtonSize): number | undefined {
+        if (!buttonSize) {
```

Comment:
May we pass default `buttonHeight` in cases if `buttonSize` is not provided? In this case we have control on button size instead of Bolt.

What do you think?

Reply:
This button also can be configured by merchant on the Bolt side. So I think in case if in some reasons we won't have configs from BC side, merchant can control this button from Bolt side, instead of overwriting by default options.